File: Prog52aLangs/Prog52a.c
#include <stdio.h>

int main()
{
    printf("Enter Length: ");
    int length;
    scanf("%d", &length);
    printf("Enter Width: ");
    int width;
    scanf("%d", &width);
    int area = length * width;
    int perim = length * 2 + width * 2;
    printf("\nThe Area is %d", area);
    printf("\nThe Perimeter is %d", perim);

    return 0;
}
/*
Enter Length: 62
Enter Width: 57

The Area is 3534
The Perimeter is 238
*/

File: Prog52aLangs/Prog52a.go
package main

import (
  "fmt"
)

func main() {
	var length int
	var width int
	var area int
	var perim int

  	fmt.Print("Enter Length: ")
	fmt.Scan(&length)
	fmt.Print("Enter Width: ")
	fmt.Scan(&width)
	area = length * width
	perim = 2 * length + 2 * width
	fmt.Printf("Area is %v\n", area)
	fmt.Printf("Perimeter is %v\n", perim)
}
/*
Enter Length: 17
Enter Width: 31
Area is 527
Perimeter is 96
*/